### PR TITLE
Support for provisioning multiple devices at once

### PR DESCRIPTION
**Pull request by adgroc**
Added capability to easily provision multiple devices via a csv/text file

**DeviceExplorer**
- added capability to provision multiple devices in an IoT hub via csv file

**iothub-explorer.js**
- added import command to provision multiple devices in an IoT hub via text file
- added createDevice function that is utilized by the create and import commands

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@adgroc__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

**Comment by azurecla** [comment]
__@adgroc__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

File: tools/iothub-explorer/iothub-explorer.js [hunk 1]
```diff
@@ -143,27 +143,23 @@ else if (command === 'list') {
...
-    if (e instanceof SyntaxError) inputError('Device information isn\'t valid JSON');
-    else throw e;
+  
+  var data = fs.readFileSync(arg1, 'utf-8');
+  
+  var deviceIds = data.split('\n');
+  
+  for(var i=0; i<deviceIds.length; i++){
+    var deviceId = deviceIds[i].replace('\r','');
+    createDevice(deviceId);
```
Comment (damonbarry) [inline tools/iothub-explorer/iothub-explorer.js]: This code looks correct, although I'd prefer to use more common (and succinct) JavaScript patterns:
```
deviceIds.forEach(function (id) {
  createDevice(id.trim());
});
```

**Comment by damonbarry** [comment]
I made one minor comment, but otherwise the iothub-explorer changes look good to me!

**Comment by tameraw** [comment]
@adgroc - We're working on adding a bulk upload feature that would allow for registration of a number of devices more efficiently and we'll introduce the change on DeviceExplorer and iothub-explorer shortly. We'll start from this code when doing so.

**Pull request closed without merging**